node/core/dispute-coordinator/src/scraping/mod.rs

```diff
@@ -270,8 +270,15 @@ impl ChainScraper {
 			Ok(unapplied_slashes) => {
 				scraped_updates.unapplied_slashes = unapplied_slashes;
 			},
-			Err(error) => {
+			Err(runtime::Error::RuntimeRequest(RuntimeApiError::NotSupported { .. })) => {
```

Comment:
I believe usages of this variant  (here and below) are useless; see paritytech/substrate#13344 and discussion

The exhaustive list of methods supported by the runtime is determined solely by the API version, no individual method checks are possible.

Reply:
I think this is okay - the error is returned from the runtime api subsystem, not from the runtime itself. The subsystem does exactly that - keeps mapping which method is available where and if a new one is called on old runtime - returns `NotSupported`.

https://github.com/paritytech/polkadot/blob/b1cc6fa14330261a305d56be36c04e9c99518993/node/core/runtime-api/src/lib.rs#L404

Reply:
Ah, you're right, my bad. Mixed up API client errors and executor-produced errors. It's okay, then.